ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -302,124 +371,267 @@ export const Homepage = withUserProfile()(class extends React.Component<
+                              <CardButton disabled={!canCreateWorkspaces}
+                                          onClick={() => navigate(['workspaces/build'])}
+                                          style={{margin: '1.9rem 106px 0 3%'}}>
+                                Create a <br/> New Workspace
+                                <ClrIcon shape='plus-circle'
+                                         style={{height: '32px', width: '32px'}}/>
+                              </CardButton>
+                            </TooltipTrigger>
+                          </div>
+                          <div style={styles.contentWrapperRight}>
+                            <a onClick={() => navigate(['workspaces'])}
+                               style={{fontSize: '14px', color: colors.primary}}>
+                              See All Workspaces</a>
+                            <div style={{marginRight: '3%', display: 'flex',
+                              flexDirection: 'column'}}>
+                              <div style={{color: colors.primary, height: '1.9rem'}}>
+                                <div style={{marginTop: '.5rem'}}>Your Last Accessed Items</div>
+                              </div>
+                              <RecentWork dark={true}/>
+                            </div>
+                          </div>
+                        </div>)
+                  ) :
+                  <Spinner dark={true} style={{width: '100%', marginTop: '5rem'}}/>}
+              </div>
+            </div>
+          </FadeBox>
           <div>
-            <div style={styles.footer}>
-              <div style={styles.footerInner}>
-                <div style={styles.footerTitle}>
-                  How to Use the All of Us Researcher Workbench</div>
-                <div style={{display: 'flex', justifyContent: 'flex-end'}}>
-                  <TooltipTrigger content='Coming Soon' side='left'>
-                    <a href='#' style={{color: colors.white}}>See all documentation</a>
-                  </TooltipTrigger>
-                </div>
-                <div style={{display: 'flex', flexDirection: 'row',
-                  width: '87.34%', justifyContent: 'space-between'}}>
-                  {footerLinks.map((col, i) => {
-                    return <React.Fragment key={i}>
-                      <div style={styles.linksBlock}>
-                        <div style={styles.footerText}>
-                          <div style={{color: colors.white, marginTop: '2%'}}>{col.title}</div>
-                          <ul style={{color: colors.secondary}}>
-                            {col.links.map((link, ii) => {
-                              return <li key={ii}>
-                                <a href='#' style={{color: colors.secondary}}>{link}</a>
-                              </li>;
-                            } )}
-                          </ul>
+            <div style={styles.quickTour}>
```

Comment:
Just to confirm this is something that was already there and there is no change to quicktour?

Reply:
Right, I didn't really change quicktour. Looks like github is a bit confused with this diff. This line is equivalent to line 350 in the original code: https://github.com/all-of-us/workbench/pull/2511/files#diff-64640e577f5207068021f36d41dfcedfL350